pkg/config/containers.conf

```diff
@@ -515,6 +515,11 @@ default_sysctls = [
 #
 #stop_timeout = 10
 
+# Number of seconds to wait before exit command in API process is given to.
+# This mimics Docker's exec cleanup behaviour, where the default is 5 minutes.
+#
+#exit_command_delay = 5 * 60
```

Comment:
I don't think this is valid toml syntax, it should just be 300.
Make sure to mention in the comment above that the value is in seconds

Reply:
Thanks for the review! I have updated the docs and changed the configuration to use the direct value and added specific mentions that the value is in seconds.

Reply:
I don't think this is an issue but maybe @mheon knows more.
As I understand it the exit command delay is only started after the exec command has exited. The delay is only used to allow API users to get information about the exec session. After the delay the information is deleted from the podman db.

Reply:
Ah okay that makes sense then. Commits have been squashed

Reply:
Shouldn't be an issue. The delay is how long results of the exec are retained in the DB after the exec finishes, so there's no relation with how long the exec call takes - it's basically how long the Podman client has to call back and ask what the exit code was.

Reply:
I think we could probably go as long as 1-2 seconds on fast systems, assuming low load; 5-10 is probably safe everywhere. This is mainly for full API compatibility, in case folks had wierd apps targeting the Docker API